### DISCUSSION TITLE
建议：根据博客标题和子标题在右边生产内容大纲

### DISCUSSION THREAD

**Discussion by jmszg** [gaowei-space/markdown-blog#20] [Ideas]
大佬的项目很喜欢，我也下载下来研究一下，建议在在博客文章的右边根据markdown的标题生成一个内容大纲，这种就极其方便别人简明扼要的阅读博文。

**Comment by gaowei-space** [comment]
谢谢支持 😜

是指这样的功能吗？或者有其他例子让我康康么？
<img width="383" alt="image" src="https://user-images.githubusercontent.com/10205742/206894221-de932a3c-667e-4e5d-ba56-4dcffe26c03a.png">

**Comment by gaowei-space** [comment]
当前 `app.go` 主逻辑不够优雅，我正在进行重构的工作~

**Comment by jmszg** [comment]
![image](https://user-images.githubusercontent.com/14332997/206894785-072aaef6-bd3a-4903-bb1a-c3d47557c8ea.png)
是的是的，类似这种的功能！

**Comment by jmszg** [comment]
根据几级标题来生成一个文章的刚要

**Comment by jmszg** [comment]
> 当前 `app.go` 主逻辑不够优雅，我正在进行重构的工作~

加油！

**Comment by gaowei-space** [comment]
> ![image](https://user-images.githubusercontent.com/14332997/206894785-072aaef6-bd3a-4903-bb1a-c3d47557c8ea.png) 是的是的，类似这种的功能！

get~

**Comment by jackloves111** [comment]
加油！
